inc/rtcan.h

```diff
@@ -122,7 +122,7 @@ typedef struct
 #define RTCAN_TX_QUEUE_SIZE      (RTCAN_TX_QUEUE_LENGTH * RTCAN_TX_QUEUE_ITEM_SIZE)
 
 #define RTCAN_RX_NOTIF_QUEUE_LENGTH     10
-#define RTCAN_RX_NOTIF_QUEUE_ITEM_SIZE  1 // one pointer = 1x ULONG
+#define RTCAN_RX_NOTIF_QUEUE_ITEM_SIZE  (sizeof(rtcan_msg_t*) / sizeof(ULONG)) // one pointer = 1x ULONG
```

Comment:
This is the right constant value, but I don't think this should be the name of the constant that users use in their code. This particular group of constants is internal to RTCAN (used to send pointers to received messages to the RTCAN Rx thread for dispatch to subscribers). 

I suggest:
```c
// user constant
#define RTCAN_RX_QUEUE_ITEM_SIZE (sizeof(rtcan_msg_t*) / sizeof(ULONG))

// internal constant
#define RTCAN_RX_NOTIF_QUEUE_ITEM_SIZE (RTCAN_RX_QUEUE_ITEM_SIZE)
```

I think we should perhaps rename all of the internal constants, or `#undef` them at the end of the header so they don't get exposed to users and potentially cause confusion. Maybe they should be prefixed with an underscore if they are internal?